src/Servers/Kestrel/test/Interop.FunctionalTests/H2SpecTests.cs

```diff
@@ -61,6 +57,14 @@ public static TheoryData<H2SpecTestCase> H2SpecTestCases
                 var dataset = new TheoryData<H2SpecTestCase>();
                 var toSkip = new string[] { /*"http2/5.1/8"*/ };
 
+                var supportsAlpn =
```

Comment:
Can you shortcircuit this and return dataset if !supportsAlpn?

Reply:
No, all the http (h2c) cases are still supported.